includes/html/print-dependency-map.inc.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+use LibreNMS\Config;
+
+if (!Auth::user()->hasGlobalRead()) {
+    $sql = 'WHERE device_id IN (SELECT device_id FROM devices_perms WHERE user_id='. Auth::id() . ')';
```

Comment:
Please use Permission class to get user devices

Reply:
Ok, but where can i find an example to get all accessed devices at once?
I don't want to check every device in a single query for access.

Reply:
Like so;
```Php
$device_ids = Permissions::devicesForUser()->toArray() ?: [0];
$sql = "WHERE `device_id` IN " .dbGenPlaceholders(count($device_ids));
$param = $device_ids;
```

However like I said in the other comment, this page should be written in laravel, and use the eloquent models functions